<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="2" passed="1" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2023-05-24T23:56:00 PET" name="Default suite" finished-at="2023-05-24T23:56:39 PET" duration-ms="39007">
    <groups>
    </groups>
    <test started-at="2023-05-24T23:56:00 PET" name="Default test" finished-at="2023-05-24T23:56:39 PET" duration-ms="39007">
      <class name="main">
        <test-method is-config="true" signature="SetUp()[pri:0, instance:main@27f981c6]" started-at="2023-05-24T23:56:00 PET" name="SetUp" finished-at="2023-05-24T23:56:09 PET" duration-ms="8745" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- SetUp -->
        <test-method signature="emptyCard()[pri:0, instance:main@27f981c6]" started-at="2023-05-24T23:56:09 PET" name="emptyCard" finished-at="2023-05-24T23:56:21 PET" duration-ms="12193" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- emptyCard -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:main@27f981c6]" started-at="2023-05-24T23:56:21 PET" name="tearDown" finished-at="2023-05-24T23:56:21 PET" duration-ms="118" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="SetUp()[pri:0, instance:main@27f981c6]" started-at="2023-05-24T23:56:21 PET" name="SetUp" finished-at="2023-05-24T23:56:27 PET" duration-ms="5991" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- SetUp -->
        <test-method signature="fillCart()[pri:0, instance:main@27f981c6]" started-at="2023-05-24T23:56:27 PET" name="fillCart" finished-at="2023-05-24T23:56:39 PET" duration-ms="11781" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Protein total sum does not match expected [2.73] but found [1.0]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Protein total sum does not match expected [2.73] but found [1.0]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1413)
at org.testng.Assert.assertEquals(Assert.java:844)
at main.fillCart(main.java:65)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:962)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:806)
at org.testng.TestRunner.run(TestRunner.java:601)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:433)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:427)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:387)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- fillCart -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:main@27f981c6]" started-at="2023-05-24T23:56:39 PET" name="tearDown" finished-at="2023-05-24T23:56:39 PET" duration-ms="117" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- main -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
